lib/web_ui/lib/src/engine/text_editing.dart

```diff
@@ -341,6 +341,9 @@ class TextEditingElement {
       _subscriptions.add(domElement.onKeyUp.listen((event) {
         _handleChange(event);
       }));
+      /// In Firefox the context menu item "Select All" does not work without
+      /// listening onSelect. On the other browersers onSelectionChange is
```

Comment:
typo: "browsers"
nit: "listening to onSelect"

Reply:
done.